frontend/src/app/main/data/style_dictionary.cljs

```diff
@@ -198,33 +198,34 @@
 
   (build-dictionary [_]
     (let [platform "json"
-          config' (clj->js config)]
-      (-> (sd. config')
-          (.buildAllPlatforms platform)
-          (p/then #(.getPlatformTokens ^js % platform))
-          (p/then #(.-allTokens ^js %))))))
-
-(defn resolve-tokens-tree+
+          config' (clj->js config)
+          build+ (-> (sd. config')
+                     (.buildAllPlatforms platform)
+                     (p/then #(.getPlatformTokens ^js % platform)))]
+      (->> (rx/from build+)
+           (rx/map #(.-allTokens ^js %))))))
```

Comment:
You could also leave the `.-allTokens` as it was before, with `p/then` and just return a `(rx/from` with the result.